services/movie_session.py

```diff
@@ -0,0 +1,56 @@
+import datetime
+from typing import Optional
+
+from db.models import models
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(
+        movie_show_time: datetime.datetime,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    movie_ = Movie.objects.get(id=movie_id)
+    hall = CinemaHall.objects.get(id=cinema_hall_id)
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=movie_,
+        cinema_hall=hall
+    )
+
+
+def get_movies_sessions(
+        session_date: str = None

```

Comment:
use typehint like: Optional[YOUR_VALUE_TYPE] = None
for example: list that contains string elements and can be None has typehint Optional[list[str]]